rasgotransforms/rasgotransforms/transforms.py

```diff
@@ -0,0 +1,113 @@
+import os
+from typing import Callable, Dict, Optional, Union
+from pathlib import Path
+import inspect
+import yaml
+
+from .jinja import render
+from .main import DataWarehouse
+
+
+class Transforms(object):
+    """
+    Dynamically adds class methods which render each transform in the package
+    """
+
+    def __init__(self):
+        transforms_dir = Path(os.path.dirname(__file__), 'transforms')
+        for transform_path in transforms_dir.rglob('*/*.yaml'):
+            with open(transform_path) as fp:
+                transform_config = yaml.safe_load(fp)
+            setattr(self.__class__, transform_config['name'], _get_render_method(transform_config))
+
+
+def _gen_method_signature(udt_func: Callable, transform_config: Dict) -> inspect.Signature:
```

Comment:
Please don't do module level `_` underscore started methods. Use `__all__ = ["Transforms"]` at the top instead. 

More details on why here: https://rasgoteam.slack.com/archives/C03A8E9PZ5G/p1654612707517819?thread_ts=1654525069.737569&cid=C03A8E9PZ5G